scripts/sommelier

```diff
@@ -67,6 +67,53 @@ init_wine() {
       mv "$WINEPREFIX" "${WINEPREFIX}.bak-$(date +%s)"
   fi
 
+  if [[ -n "${SOMMELIER_INSTALL_MONO:-}" ]]; then
+    if [[ $(grep -c mscoree "$WINEPREFIX"/user.reg 2&>/dev/null) -eq 0 ]]; then
```

Comment:
if wineprefix has dotnet20 installed this still equals 0 and then re downloads mono we have to place more checks for this.